index.html

```diff
@@ -519,6 +519,79 @@ <h3>LinkedDataSignature2016</h3>
       </section>
 
+
+      <section id="MerkleProof2019" about="https://w3id.org/security#MerkleProof2019">
+
+        <h3>MerkleProof2019</h3>
+          <p>
+A Linked Data signature is used for digital signatures on RDF Datasets.
+The default canonicalization mechanism is specified in the RDF Dataset
+Normalization specification, which effectively deterministically names all
+unnamed nodes. The default signature mechanism
+uses a SHA-256 digest and ECDSA to perform the digital signature. More information at
+              <a href="https://w3c-ccg.github.io/lds-merkle-proof-2019/">https://w3c-ccg.github.io/lds-merkle-proof-2019/</a>
+          </p>
+          <dl>
+            <dt>Status</dt>
+            <dd property="vs:term_status">stable</dd>
+            <dt>Parent Class</dt>
+            <dd>LinkedDataSignature</dd>
+            <dt>Expected properties</dt>
+            <dd>proofValue</dd>
+            <dt>Signature Properties</dt>
+            <dd>
+              <dl>
+                <dt>Default Canonicalization Algorithm</dt>
+                <dd>
+<a rel="canonicalizationAlgorithm"
+  href="https://w3id.org/security#GCA2015">https://w3id.org/security#GCA2015</a>
+                </dd>
+                <dt>Default Signature Algorithm</dt>
+                <dd>
+<a rel="signatureAlgorithm"
+  href="http://www.w3.org/2000/09/xmldsig#rsa-sha256">
+http://www.w3.org/2000/09/xmldsig#rsa-sha256</a>
+                </dd>
+              </dl>
+            </dd>
+          </dl>
+          <p>
+The example below shows how a basic JSON-LD signature is expressed in a
+JSON-LD snippet. Note that the proof property is directly embedded in the
+object. The signature algorithm specifies how the signature can be generated
+and verified.
+          </p>
+          <pre class="example prettyprint language-jsonld">
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://www.blockcerts.org/schema/3.0-alpha/context.json",
+    "https://www.w3.org/2018/credentials/examples/v1"
+  ],
+  "id": "urn:uuid:bbba8553-8ec1-445f-82c9-a57251dd731c",
+  "type": [
+    "VerifiableCredential",
+    "BlockcertsCredential"
+  ],
+  "issuer": "did:example:23adb1f712ebc6f1c276eba4dfa",
+  "issuanceDate": "2010-01-01T19:33:24Z",
+  "credentialSubject": {
+    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+    "alumniOf": {
+      "id": "did:example:c276e12ec21ebfeb1f712ebc6f1"
+    }
+  },
+  "proof": {
+    "type": "MerkleProof2019",
+    "created": "2020-10-15T15:16:52.191052",
+    "proofValue": "zMcm4LfQFUZkWZyLpVrzYzaHRssD3xGQ6Yh6nEhErjfAqnxDWo7gBawdhynnbiucoqqwye54oTs5p6frH385nVvBf2UXMgRVtBZLq6tHN1MY1hQBMBBtXbWbYzj99GotAJRs7PxcvQmUQKHs8hkokZhgw92CNAtvDn2jeycvuyFPtWorxZaGP23mDkgwpberKmAqcPcsUFNooGe1UkQj8wws8D9pdpAVmuKf5Zo78iq1tUWSUdGwN9sK57SQP6YTtL59b7WUK1NEHD1GL8o5Z2mY6MnxuxfiqkpjepaPVqTPfgmaCuS",
+    "proofPurpose": "assertionMethod",
+    "verificationMethod": "ecdsa-koblitz-pubkey:0x7e30a37763e6Ba1fFeDE1750bBeFB4c60b17a1B3"
```

Comment:
You may want to use a multibase encoded multikey value here... this needs to be, at a minimum, a URI -- don't think ecdsa-koblitz-pubkey is a URI? Can this value be expressed as a DID? That would be best.

Reply:
`"verificationMethod": "ecdsa-koblitz-pubkey:0x7e30a37763e6Ba1fFeDE1750bBeFB4c60b17a1B3"` this is very poorly supported form 2016.... see

https://github.com/w3c-ccg/lds-jws2020

https://w3c-ccg.github.io/lds-ecdsa-secp256k1-2019/

Reply:
also thats an ethereum address not a public key.

Reply:
See also: https://github.com/decentralized-identity/EcdsaSecp256k1RecoverySignature2020

Reply:
Yeah at this point this is a configuration in my local `[cert-issuer](https://github.com/blockchain-certificates/cert-issuer/)`, that I have probably taken from @antonyronning. Again this does not effect immediately the proofing mechanism as this is a user set parameter, but if you find that confusing, again I can try and amend the example, no problem.

Reply:
I recommend making `verificationMethod` a URI or DID.

Reply:
Agreed. At present, there is no such URI scheme called ecdsa-koblitz-pubkey, making this value questionable as the expectation is that it should be a URI of some kind.

Reply:
We are using DID here, and it points directly to the key within that DID (as 1 DID can have many keys).

Since DID has a form of URN, can it be generalized to be a URN?

Reply:
It's supposed to be a URL (you need to be able to retrieve the key information using the `verificationMethod` parameter and use that to verify the signature).

If you use a DID here, we're all good, except that the example is wrong :).

Can you propose a change to the example so it uses a DID instead of this weird `"ecdsa-koblitz-pubkey:0x7e30a37763e6Ba1fFeDE1750bBeFB4c60b17a1B3"` thing?

Reply:
One possible solution would be to use did:key here... its functioanlly the same (but uses a public key instead of an ethereum address):

https://github.com/transmute-industries/did-key.js/blob/master/packages/test-vectors/src/did-core-conformance/secp256k1/secp256k1.json#L43

the verificationMethod would be:

`did:key:zQ3shU9mxTYfMTLceuppoNM5siowHvw3xBk21r855kwgNHNAX#zQ3shU9mxTYfMTLceuppoNM5siowHvw3xBk21r855kwgNHNAX`

Reply:
Here is how we are using:
```
{
  ...
  verificationMethod: "did:vaultie:0x8b9ac093ce03ee542bd4a37d3df77ad33079419f#keys-1"}
```

Reply:
Yes, that's good, please update example to reflect that.